### PR TITLE
change to the `.format()` function

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
@MasterofJOKers could you review?

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{0}:{1}'.format(module, call)]
```
Comment (MasterofJOKers) [inline tests/utils.py]: weren't we going to use `{}` instead of `{0}`? You did use `{}` in you last Pull-Request.

Comment (MarkusHackspacher) [inline tests/utils.py]: With Python 2.6 is working `'{}'.format('a')`, but not `'{}:{}'.format('a', 'b')`. You have to use `'{0}:{1}'.format('a', 'b')`. But better shoud be `'{module}:{call}'.format(module='a', call='b')`.

Comment (MasterofJOKers) [inline tests/utils.py]: We don't support Python2.6 any more.

Comment (MasterofJOKers) [inline tests/utils.py]: And I don't think we need `{module}:{call}`, as it's only testing code that will not get translated and it's pretty obvious, that this is just a `':'.join([module, call])`.

Comment (squiddy) [inline tests/utils.py]: Yeah, `"{module}:{call}".format(module=module, call=call)` is definitly too much. And also, that's a good way to convince me to keep `"%s:%s" % (module, call)`. :D

But `"{}:{}".format(module, call)` is fine I guess.

File: tests/utils.py [hunk 2]
```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{}:{}'.format(module, call)]
```
Comment (MarkusHackspacher) [inline tests/utils.py]: Done

**Pull request merged**